src/app/modules/browse/browse-controller.js

```diff
@@ -140,9 +140,29 @@
             }
             return _localize(message, params);
         }
+        function setHiddenBooksMessages() {
+            if ($scope.allLicenses) {
```

Comment:
I agree about the localization issue, but I was trying to minimize the changes from what was already there.